WebDriverAgentLib/Categories/XCUIDevice+FBHelpers.m

```diff
@@ -339,6 +339,28 @@ - (BOOL)fb_setAppearance:(FBUIInterfaceAppearance)appearance error:(NSError **)e
     [invocation invoke];
     return YES;
   }
+
+#if __clang_major__ >= 14 && __clang_minor__ >= 0 && __clang_patchlevel__ >= 3
```

Comment:
I assume the original issue was that the getAppearance call returns a false value

Reply:
The clang thing is to compile `self.appearance` which is available since Xcode 14.3